2024-04-02 18:56:40,450 INFO    MainThread:290114 [wandb_setup.py:_flush():75] Loading settings from /home/presta/.config/wandb/settings
2024-04-02 18:56:40,450 INFO    MainThread:290114 [wandb_setup.py:_flush():75] Loading settings from /scratch/DVC-PyTorch/wandb/settings
2024-04-02 18:56:40,451 INFO    MainThread:290114 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2024-04-02 18:56:40,451 INFO    MainThread:290114 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'main_dvc.py', 'program': 'main_dvc.py'}
2024-04-02 18:56:40,489 INFO    MainThread:290114 [wandb_init.py:_log_setup():386] Logging user logs to /scratch/DVC-PyTorch/wandb/run-20240402_185640-3v9dn20g/logs/debug.log
2024-04-02 18:56:40,526 INFO    MainThread:290114 [wandb_init.py:_log_setup():387] Logging internal logs to /scratch/DVC-PyTorch/wandb/run-20240402_185640-3v9dn20g/logs/debug-internal.log
2024-04-02 18:56:40,527 INFO    MainThread:290114 [wandb_init.py:init():420] calling init triggers
2024-04-02 18:56:40,528 INFO    MainThread:290114 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {'log': '', 'pretrain': '', 'test': False, 'testhevc': False, 'testvtl': False, 'testmcl': False, 'testauc': False, 'rerank': False, 'allpick': False, 'config': 'DVC_1024.json'}
2024-04-02 18:56:40,528 INFO    MainThread:290114 [wandb_init.py:init():471] starting backend
2024-04-02 18:56:40,528 INFO    MainThread:290114 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-02 18:56:40,588 INFO    MainThread:290114 [backend.py:ensure_launched():219] starting backend process...
2024-04-02 18:56:40,602 INFO    MainThread:290114 [backend.py:ensure_launched():225] started backend process with pid: 290177
2024-04-02 18:56:40,603 INFO    MainThread:290114 [wandb_init.py:init():480] backend started and connected
2024-04-02 18:56:40,636 INFO    MainThread:290114 [wandb_init.py:init():550] updated telemetry
2024-04-02 18:56:40,763 INFO    MainThread:290114 [wandb_init.py:init():581] communicating current version
2024-04-02 18:56:41,542 INFO    MainThread:290114 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.16.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-02 18:56:41,543 INFO    MainThread:290114 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2024-04-02 18:56:41,757 INFO    MainThread:290114 [wandb_init.py:init():624] starting run threads in backend
2024-04-02 18:56:42,721 INFO    MainThread:290114 [wandb_run.py:_console_start():1827] atexit reg
2024-04-02 18:56:42,724 INFO    MainThread:290114 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2024-04-02 18:56:42,724 INFO    MainThread:290114 [wandb_run.py:_redirect():1706] Redirecting console.
2024-04-02 18:56:42,726 INFO    MainThread:290114 [wandb_run.py:_redirect():1762] Redirects installed.
2024-04-02 18:56:42,727 INFO    MainThread:290114 [wandb_init.py:init():651] run started, returning control to user process
2024-04-02 18:56:49,443 INFO    MainThread:290114 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2024-04-02 18:56:49,445 INFO    MainThread:290114 [wandb_run.py:_restore():1769] restore
2024-04-02 18:56:52,061 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 737
}

2024-04-02 18:56:52,308 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 737
}

2024-04-02 18:56:52,577 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 737
}

2024-04-02 18:56:53,131 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 4305
}

2024-04-02 18:56:53,234 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 4989
}

2024-04-02 18:56:53,338 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 4989
}

2024-04-02 18:56:53,440 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}

2024-04-02 18:56:53,543 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}

2024-04-02 18:56:53,647 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}

2024-04-02 18:56:53,750 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}

2024-04-02 18:56:53,853 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}

2024-04-02 18:56:54,319 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}

2024-04-02 18:56:54,751 INFO    MainThread:290114 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4989
  total_bytes: 4989
}
local_info {
}

2024-04-02 18:56:55,926 INFO    MainThread:290114 [wandb_run.py:_append_files():2194] logging synced files
